### PR TITLE
Disable fast livesync with hook

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
Add before livesync hook to be able to control the livesync data. Added property in livesync service to force full sync which will be passed to the livesync data.

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: services/livesync-service-base.ts [hunk 1]
```diff
@@ -87,7 +87,11 @@ class LiveSyncServiceBase implements ILiveSyncServiceBase {
 								return;
 							}
 
-							data.canExecuteFastSync = that.$liveSyncProvider.canExecuteFastSync(filePath, data.platform);
+							if (data.forceExecuteFullSync) {
+								data.canExecuteFastSync = false;
```
Comment (rosen-vladimirov) [inline services/livesync-service-base.ts]: you can remove the `if` and `else` and use:
```TypeScript
data.canExecuteFastSync = data.forceExecuteFullSync ? false : that.$liveSyncProvider.canExecuteFastSync(filePath, data.platform);
```
But not a merge stopper

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by rosen-vladimirov** [comment]
Please retarget to release branch

**Comment by TsvetanMilanov** [comment]
Closing and retarget to release.

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request closed without merging**